src/Infiltration/ui/SlashGame.tsx

```diff
@@ -70,7 +73,7 @@ export function SlashGame({ difficulty: _difficulty, onSuccess, onFailure }: IMi
         {hasAugment ? (
           <Box sx={{ my: 1 }}>
             <Typography variant="h5">Guard will drop in...</Typography>
-            <GameTimer millis={guardingTimeRef.current} onExpire={() => null} ignoreAugment_WKSharmonizer noPaper />
+            <GameTimer millis={guardingTime} onExpire={() => null} ignoreAugment_WKSharmonizer noPaper />
```

Comment:
Not convinced that ref is the answer here.

Both guardingTime and hasAugment are:
1. Used during rendering
2. Should not be recalculated multiple times for the same instance of the game (not a big deal for hasAugment)

They should not be useRef because of point 1. From the react documentation: `useRef is a React Hook that lets you reference a value that’s not needed for rendering.`

There is also a pitfall section, stating: `Do not write or read ref.current during rendering.` We are reading the values of both of these during rendering. Since the current value of both of these is needed during rendering and not just in effects and event listeners, they should not be refs.

Since they should not be recalculated every render, they should not be plain variables inside the function body, they need to be kept stable between renders in some manner. Since refs are off the table, the remaining options are useMemo and useState. State might make more sense here, with the game initializer useEffect performing the state update. timingWindow can be entirely contained within the useEffect. I'll post an updated commit.